firefox-ios/Client/Frontend/Reader/ReaderModeHandlers.swift

```diff
@@ -90,9 +94,9 @@ struct ReaderModeHandlers: ReaderModeHandlersProtocol {
     }
 
     private static func generateHtmlFor(readabilityResult: ReadabilityResult,
+                                        style: consuming ReaderModeStyle,
                                         profile: Profile) -> GCDWebServerDataResponse? {
-        var readerModeStyle = ReaderModeStyle.defaultStyle()
-
+        var readerModeStyle = consume style
```

Comment:
The use of consume here seems superfluous. `ReaderModeStyle` is a struct and it will go out of scope within the bounds of `generateHtmlFor` regardless. What effect are you trying to achieve?

Reply:
Basically I want to avoid additional copies.

Reply:
I don't think that is a concern here given it will go out of scope with the function. Can you remove please?